sympy/printing/latex.py

```diff
@@ -87,10 +87,10 @@
     'scr': lambda s: r'\mathscr{'+s+r'}',
     'frak': lambda s: r'\mathfrak{'+s+r'}',
     # Brackets
-    'norm': lambda s: r'\left\lVert{'+s+r'}\right\rVert',
+    'norm': lambda s: r'\left\|{'+s+r'}\right\|',
```

Comment:
No it should not.  `\lVert` and `\` give two bars.  (`\lvert` and `|` give one.)